src/TestSuite/FixtureSchemaExtension.php

```diff
@@ -67,6 +79,6 @@ public function executeBeforeTest(string $test): void
      */
     public function executeAfterTest(string $test, float $time): void
     {
-        $this->state->afterTest();
+        $this->state->afterTest($test);
```

Comment:
fyi, this hook provides the full name of the class::method as $test.

Reply:
Good point. I've updated the docstrings.